res/controllers/midi-components-0.0.js

```diff
@@ -766,7 +766,7 @@
     };
 
     JogWheelBasic.prototype = new Component({
-        vinylMode: true,
+        _vinylMode: true, // private, accessible via setters defined below
```

Comment:
Is this change backward compatible? Wasn't it possible before to read/write the member vinylControl from a mapping script?

Reply:
yeah, its backwards compatible because the setter/getters defined below "emulate" that property. See 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set

Reply:
fwiw I tested this with my NS6II without making any changes in its script and it works well.